salt/utils/process.py

```diff
@@ -1084,6 +1098,90 @@ def _handle_signals(self, signum, sigframe):
+                                                fd_.readline()
+                                            ).rstrip()
+                                        )
+                                    except ValueError:
+                                        # Lock file is empty, set pid to 0 so it evaluates as False.
+                                        file_pid = 0
+                                    try:
+                                        file_mach_id = (
+                                            salt.utils.stringutils.to_unicode(
+                                                fd_.readline()
+                                            ).rstrip()
+                                        )
+                                    except ValueError:
+                                        # Lock file is empty, set mach_id to 0 so it evaluates as False.
+                                        file_mach_id = 0
+
+                            if cur_pid == file_pid:
+                                if mach_id != file_mach_id:
+                                    if not file_mach_id:
+                                        msg = (
+                                            f"gitfs lock file for pid '{file_pid}' does not "
+                                            "contain a machine id, deleting lock file which may "
+                                            "affect if using multi-master with shared gitfs cache, "
+                                            "the lock may have been obtained by another master "
+                                            "recommend updating Salt version on other masters to a "
+                                            "version which insert machine identification in lock a file."
+                                        )
+                                        log.debug(msg)
+                                        file_del_list.append(
+                                            (file_name, file_pid, file_mach_id)
+                                        )
+                                else:
+                                    file_del_list.append(
+                                        (file_name, file_pid, file_mach_id)
+                                    )
+
+                        except FileNotFoundError:
+                            log.debug("gitfs lock file: %s not found", file_name)
+
+                        for file_name, file_pid, file_mach_id in file_del_list:
+                            try:
+                                os.remove(file_name)
+                            except OSError as exc:
+                                if exc.errno == errno.ENOENT:
+                                    # No lock file present
+                                    msg = f"SIGTERM clean up of resources attempted to remove lock file {file_name}, pid '{file_pid}', machine identifier '{mach_id}' but it did not exist, exception : {exc} "
```

Comment:
Have a look at all of the long lines to wrap please, I did not mark all of them.

Reply:
done